internal/provider/data_source_http.go

```diff
@@ -55,6 +57,18 @@ your control should be treated as untrustworthy.`,
 				Computed:    true,
 			},
 
+			"ca_certificate": {
+				Description: "PEM-encoded root certificate bundle for TLS authentication",
```

Comment:
Additional to comment above, would be nice to take a hint from the TLS provider, and have the actual RFC that defines the PEM format, linked here.